docs/features.md

```
@@ -413,3 +413,65 @@ PROXY_INJECT_STREAM_USAGE=true   # default: true
+
+**Config overrides:**
+```env
+# Disable a feature
+FEATURE_TELEGRAM_BOT=disabled
+
+# Promote a feature to stable
+FEATURE_ASYNC_AGENT_JOBS=stable
+```
+
+**Admin API:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)